app/backend/src/clj/teet/integration/integration_email.clj

```diff
@@ -58,13 +61,10 @@
              "\r\n\r\n"
              (->b64 body))))))
 
-(defn- send-email!*
-  "Invoke email sending"
-  [msg]
-  (aws/invoke
-   @email
-   {:op :SendRawEmail
-    :request {:RawMessage {:Data (.getBytes (raw-message msg) "UTF-8")}}}))
+(defn send-email-smtp!* [msg]
+  (println (str "email server " email-server))
+  (println (str "message " msg))
+  (println (postal.core/send-message email-server msg)))
```

Comment:
I realise this is WIP code but remember to eventually ensure that hostile characters from user entered fields are handled safely or that the postal lib has robust handling for them, to guard against eg header injection.

Reply:
the sending emails is automated (mostly). So, the texts are from our code, just user name (which has already sanitized) comes from user. Do you have any particular use case that can be harmful and require additional protection?

Reply:
I had a quick look and at least subject comes from user supplied data sometimes. We also should be robust against future uses of this code and not rely too much on what we assume is already sanitized and what isn't.

Reply:
it is sanitized https://github.com/drewr/postal/blob/cfdec2deca126733bd3106792b753c566495685e/src/postal/sendmail.clj#L63